scripts/apps/search/MultiImageEdit.ts

```diff
@@ -150,10 +154,45 @@ export function MultiImageEditController(
             copyrightholder: compare('copyrightholder'),
             usageterms: compare('usageterms'),
             copyrightnotice: compare('copyrightnotice'),
-            extra: compare('extra'),
+            extra: compareExtra(),
             language: compare('language'),
             creditline: compare('creditline'),
         };
+
+    }
+
+    function compareExtra() {
```

Comment:
Could you add a comment why this one needs special handling and what are you doing in every loop?